docs/profiling.md

```diff
@@ -41,6 +41,27 @@ or if you're using Google Cloud:
 $ gcloud compute ssh <machine-name> -- -L 9001:127.0.0.1:9001
 ```
 
+### Manual capture
+
+Instead of capturing traces programmatically using `jax.profiler.trace`, you can
+instead start a profiling server in the script of interest by calling
+`jax.profiler.start_server(<port>)`. If you only need the profiler server to be
+active for a portion of your script, you can shut it down by calling
+`jax.profiler.stop_server()`.
+
+Once the script is running and after the profiler server has started, we can
+manually capture an trace by running:
+```bash
+$ python -m jax.collect_profile <port> <duration_in_ms>
+```
+
+By default, the resulting trace information is dumped into a temporary directory
+but this can be overridden by passing in `--log_dir=<directory of choice>`. 
```

Comment:
Mention how to open the trace in TensorBoard using the log directory, or link to the TensorBoard section.

Reply:
Done.